config.py

```diff
@@ -30,6 +33,9 @@ class Config(object):
     PASSWORD_MIN_LENGTH = 8
     PASSWORD_MAX_LENGTH = 160
 
+    OAUTH_URL = os.getenv('OAUTH_URL')
+    PARTY_URL = os.getenv('PARTY_URL')
```

Comment:
This should have a party_username and party_password for authentication.

Reply:
@gemmai95 is that the same for all services?

Reply:
Yes. Another one of these config decisions not documented unfortunately.

Reply:
@gemmai95 Is that because each service may have its own username and password for API auth now?

Reply:
Yes